### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by Lloyd-K**
Content updates for https://dfedigital.atlassian.net/browse/CPDRP-621?atlOrigin=eyJpIjoiZDNhYmE2MjEwMGVhNGRlZDk0ODc4ODMxYWE0Y2Q5ZmMiLCJwIjoiaiJ9

### CODE REVIEW

File: app/views/sprint-17/teacher-or-mentors/cip-validation/dqt-contact.html [hunk 1]
```diff
@@ -11,14 +11,22 @@
...
+      <p class="govuk-body">Sign in to the <a class="govuk-link" href="https://teacherservices.education.gov.uk/SelfService/Login"> Teacher Self Service Portal</a>.</p> 
+
+      <p class="govuk-body">You'll need to upload one of the following documents as evidence of your change of name:</p>
+
       <ul class="govuk-list govuk-list--bullet">
       <li>marriage: copy of your marriage certificate</li>
       <li>civil partnership: copy of your civil agreement certificate</li>
       <li>divorce: copy of your decree absolute and your birth certificate</li>
       <li>statutory declaration: copy of the statutory declaration</li>
-      <li>deed poll: copy of your deed poll. Once you have clicked submit, you will be taken to the Change of Details request page.</li>
```
Comment (etheya) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/dqt-contact.html:20]: Need to remove fullstop in a list item.

File: app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html [hunk 2]
```diff
@@ -11,8 +11,7 @@
 
       {{ macroPageHeader.pageHeader(pageHeading,pageSection) }}
 
-      <p class="govuk-body"><p>Your name must match the name held in the Teaching Regulation Agency records to confirm you are eligible for training and support.</p>
-      <p>If it doesn't match, you cannot continue with your training and support.</p>
+      <p class="govuk-body"><p>Your name must match the Teaching Regulation Agency records. We use these to check you’re eligible for this training and support.</p>
```
Comment (etheya) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html:14]: Looks like too many </p> tags?

File: app/views/sprint-17/teacher-or-mentors/cip-validation/particpant-email.html [hunk 3]
```diff
@@ -23,7 +23,7 @@
   {% if data['participantType'] == "participantECT" %}
 
   <p><strong>From: </strong>Department for Education: Teacher Continuing Professional Development </p>
-  <p><strong>Subject: </strong>Get set up to start your early career teacher training programme </p>
+  <p><strong>Subject: </strong>Add your details to start your early career teacher training programme</p>
```
Comment (etheya) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/particpant-email.html:26]: This is technically wrong using <strong> but this looks like it was there before - so i will fix it at some point

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html [hunk 4]
```diff
@@ -13,41 +13,45 @@
...
-      <li>enter qts.enquiries@education.gov.uk as the recipient</li>
-      <li>input the ‘Subject’ of the email as: Undertaking training and support, do not have a TRN and require a number to be issued</li>
-      <li>include the following information in your email:</li>
+      <p class="govuk-body">Follow these steps to send your documents through Galaxkey:</p>
+      <ol class="govuk-list govuk-list--number">
+      <li>Select 'Compose' to create a new secure email.</li>
+      <li>Enter 'qts.enquiries@education.gov.uk' as the recipient.</li>
+      <li>Enter ‘TRN request for early career teacher training’ as the subject line.</li>
+      <li>Include the following information in your email:</li>
         <ul>
```
Comment (etheya) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:33]: Missing the code for the list items - ?

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html [hunk 5]
```diff
@@ -13,41 +13,45 @@
...
-        <li>your full legal name (we will use this to create a record for you)</li>
+        <li>your full legal name</li>
         <li>your date of birth</li>
-        <li>an email address for us to respond to you at</li>
-        <li>attach a copy of one of the following types of identification as a file to accompany your submission. Note the restrictions on file size in the provided guidance:</li>
-          <ul>
+        <li>an email address for the team to reply to</li>
+        </ul>
+      <li>Attach a scanned copy or photo of one of the following types of ID:</li>
+            <ul>
```
Comment (etheya) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:39]: Same as above missing the code for the list items

**Review by etheya — CHANGES_REQUESTED** [review]

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html (hunk 4 above)
Comment (Lloyd-K) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:33]: It's a list within a list and seemed to render correctly on local?

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html (hunk 5 above)
Comment (Lloyd-K) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:39]: Same comment as above

File: app/views/sprint-17/teacher-or-mentors/cip-validation/dqt-contact.html (hunk 1 above)
Comment (Lloyd-K) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/dqt-contact.html:20]: That's the old content, it's been overwritten

File: app/views/sprint-17/teacher-or-mentors/cip-validation/name-not-updated.html [hunk 6]
```diff
@@ -11,12 +11,7 @@
 
       {{ macroPageHeader.pageHeader(pageHeading,pageSection) }}
 
-      <p class="govuk-body"><p>If your name has not been updated and does not match the Teaching Regulation Agency records, you have three options:</p>
-      <ul class="govuk-list govuk-list--bullet">
-      <li>use your previous name to register with the service</li>
-      <li>use your previous name to register, and then change it on the Teaching Regulation Agency records at a later date</li>
-      <li>change your name on the Teaching Regulation Agency records first and then return to register for training and support</li>
-      </ul>
+      <p class="govuk-body"><p>We cannot check your eligibility if your name is different to the one listed in Teaching Regulation Agency records.</p>
```
Comment (ethanmills) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/name-not-updated.html:14]:
```suggestion
      <p class="govuk-body">We cannot check your eligibility if your name is different to the one listed in Teaching Regulation Agency records.</p>
```

File: app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html (hunk 2 above)
Comment (etheya) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html:14]: 14

Comment (ethanmills) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html:14]:
```suggestion
      <p class="govuk-body">Your name must match the Teaching Regulation Agency records. We use these to check you’re eligible for this training and support.</p>
```

Comment (etheya) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html:14]: <p class="govuk-body"><p>Your .......

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html (hunk 4 above)
Comment (etheya) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:33]: Should still have it - but its ok for now.

File: app/views/sprint-17/teacher-or-mentors/cip-validation/sign-in-check-email-ect.html [hunk 7]
```diff
@@ -12,12 +12,12 @@
...
 
 
-      <p class="govuk-body">If you have an account, we have sent you a link to sign in.</p>
+      <p class="govuk-body">If you have an account, we've sent you a link to sign in.</p>
       <p class="govuk-body">This sign-in link will only work for this service. We do not use DfE Sign-In.</p>
       <h2 class="govuk-heading-m">If you do not get an email to sign in</h2>
       <ul class="govuk-list govuk-list--bullet">
-        <li>you must be nominated or sent sign-in details to access this service <a class="govuk-link--no-visited-state" href="/sprint-17/teacher-or-mentors/cip-validation/sign-in-check-account-ect">find out how to get an account</a>
-        <li>check with your induction tutor that they have added your details</li>
+        <li>you must be nominated or sent sign-in details to access this service - <a class="govuk-link--no-visited-state" href="/sprint-17/teacher-or-mentors/cip-validation/sign-in-check-account-ect">find out how to get an account</a>
```
Comment (ethanmills) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/sign-in-check-email-ect.html:19]:
```suggestion
        <li>you must be nominated or sent sign-in details to access this service - <a class="govuk-link--no-visited-state" href="/sprint-17/teacher-or-mentors/cip-validation/sign-in-check-account-ect">find out how to get an account</a></li>
```

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html (hunk 4 above)
Comment (ethanmills) [inline app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:33]:
```suggestion
        <ul class="govuk-list govuk-list--bullet">
```

File: app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html (hunk 5 above)
Comment (ethanmills) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/trn-get.html:39]:
```suggestion
            <ul class="govuk-list govuk-list--bullet">
```

**Review by ethanmills — COMMENTED** [review]
I'm afraid you can't send me a link to a review and expect me not to look at it 😛

File: app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html (hunk 2 above)
Comment (Lloyd-K) [reply app/views/sprint-17/teacher-or-mentors/cip-validation/name-change-ect.html]: Extra p tag removed, thanks

**Review by etheya — APPROVED** [review]

**Pull request merged**